argo_jupyter_scheduler/executor.py

```diff
@@ -573,3 +605,34 @@ def create_job_record(
 
         session.add(job)
         session.commit()
+
+
+def get_slack_token_channel(envs):
+    token = None
+    channel = None
+    for env in envs:
```

Comment:
Instead of looping over all the environment variables, can we use `parameters.get("SLACK_TOKEN")`?